src/uikit.cs

```diff
@@ -11254,8 +11553,12 @@ interface UISearchDisplayDelegate {
 	[BaseType (typeof (NSObject))]
 	partial interface UISearchResultsUpdating {
 		[Abstract]
-	    [Export ("updateSearchResultsForSearchController:")]
-	    void UpdateSearchResultsForSearchController (UISearchController searchController);
+		[Export ("updateSearchResultsForSearchController:")]
+		void UpdateSearchResultsForSearchController (UISearchController searchController);
+
+		[TV (14,0), NoWatch, NoiOS]
+		[Export ("updateSearchResultsForSearchController:selectingSearchSuggestion:")]
+		void SelectingSearchSuggestion (UISearchController searchController, IUISearchSuggestion searchSuggestion);
```

Comment:
wrong part of the selector used for the name
it's an overload to the existing one, with an extra parameter